test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -1022,19 +1024,22 @@ function flux(::Energy, ::SGSFlux, atmos, args)
         FT(1 // 2) *
         ((gm.ρu[1] * ρ_inv)^2 + (gm.ρu[2] * ρ_inv)^2 + (gm.ρu[3] * ρ_inv)^2)
     e_tot_up = ntuple(i -> total_energy(e_kin[i], _grav * z, ts_up[i]), N_up)
```

Comment:
I am confused here. It seems like `e_kin` is defined in line 1021 as a scalar, but here we are using index [i]. I think this is an error. I think we should define the tuple e_kin_up using the updraft vertical velocity instead of the grid-mean vertical velocity, and apply this definition here.